### ISSUE TITLE
Just a question - Re: Amazon Fire / Android

### ISSUE THREAD

**Issue by rgrokett** [chanlawrencet/SmartDisplay#4]
Just a question, are you using a rooted Amazon Fire running Android & Google Play?  
Or did you just install Google Play store onto the Fire, then Expo App for Android  and haul the yarn packaged app in that way?  

Thanks!
Russell

**Comment by chanlawrencet** [comment]
Hi Russell,

Just a regular Amazon Fire! I enabled sideloading apps (which should be in settings) and then downloaded the Expo apk (from like apk mirror or something like that) for debugging. Then, I used Expo to build the app and after copying it over via usb, I was able to install it (same way as sideloading and from the file explorer. Hope this helps -- let me know if there's anything I can clarify or if you want links to other instructions :)

Cheers
